<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropLight</class>
 <widget class="QWidget" name="PropLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>349</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>349</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PropLight</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Light &amp;type:</string>
       </property>
       <property name="buddy">
        <cstring>LightType</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="LightType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Diffuse color:</string>
       </property>
       <property name="buddy">
        <cstring>DiffuseColor</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="Armed::ColorButton" name="DiffuseColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Spec&amp;ular color:</string>
       </property>
       <property name="buddy">
        <cstring>SpecularColor</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="Armed::ColorButton" name="SpecularColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="LabelAttn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Attenuation:</string>
       </property>
       <property name="buddy">
        <cstring>Range</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="LabelRange">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Range:</string>
       </property>
       <property name="buddy">
        <cstring>Range</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="Range">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="LabelConstant">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Constant:</string>
       </property>
       <property name="buddy">
        <cstring>AttnConstant</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="AttnConstant">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="LabelLinear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Linear:</string>
       </property>
       <property name="buddy">
        <cstring>AttnLinear</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="AttnLinear">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="LabelQuadratic">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Quadratic:</string>
       </property>
       <property name="buddy">
        <cstring>AttnQuadratic</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="AttnQuadratic">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="3">
      <widget class="QLabel" name="LabelSpot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Spotlight &amp;cone:</string>
       </property>
       <property name="buddy">
        <cstring>InnerAngle</cstring>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="LabelInner">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Inner angle:</string>
       </property>
       <property name="buddy">
        <cstring>InnerAngle</cstring>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="InnerAngle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="LabelOuter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Outer angle:</string>
       </property>
       <property name="buddy">
        <cstring>OuterAngle</cstring>
       </property>
      </widget>
     </item>
     <item row="14" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="OuterAngle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="LabelFalloff">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Falloff:</string>
       </property>
       <property name="buddy">
        <cstring>Falloff</cstring>
       </property>
      </widget>
     </item>
     <item row="15" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="Falloff">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="Line" name="LineAttn">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="Line" name="LineSpot">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QCheckBox" name="CastShadows">
       <property name="text">
        <string>Cast shadows</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Armed::ColorButton</class>
   <extends>QPushButton</extends>
   <header>ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LightType</tabstop>
  <tabstop>DiffuseColor</tabstop>
  <tabstop>SpecularColor</tabstop>
  <tabstop>CastShadows</tabstop>
  <tabstop>Range</tabstop>
  <tabstop>AttnConstant</tabstop>
  <tabstop>AttnLinear</tabstop>
  <tabstop>AttnQuadratic</tabstop>
  <tabstop>InnerAngle</tabstop>
  <tabstop>OuterAngle</tabstop>
  <tabstop>Falloff</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
